common/src/main/java/org/wildfly/httpclient/common/HttpServerHelper.java

```diff
@@ -38,11 +35,11 @@ private HttpServerHelper() {
 
     }
 
-    public static void sendException(HttpServerExchange exchange, int status, Throwable e) {
+    public static void sendException(HttpServerExchange exchange, HttpServiceConfig serviceConfig, int status, Throwable e) {
         try {
             exchange.setStatusCode(status);
             exchange.getResponseHeaders().put(Headers.CONTENT_TYPE, "application/x-wf-jbmar-exception;version=1");
```

Comment:
@ropalka shouldn't this be version 2?

Reply:
@ropalka just so you know it, your feedback here is still pending. I'm afraid this line is wrong in the HttpServerHelper and it should be using version 2, but you will know better than I do if this is really wrong.

Reply:
IMO this is version 1 of jboss marshalling because Stuart wasn't able to introduce versioning in the protocol.
Setting it to version 1 ensured backward compatibility. But with this PR it is possible to change that value for jakarta namespace @fl4via